### PR TITLE
Implement new styling for Pagoda Docs

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Closes: https://pagodaplatform.atlassian.net/browse/DEC-717

### CODE REVIEW

**Review by bucanero — APPROVED** [review]
@shelegdmitriy should I merge this PR? let me know in case you're still working on it

**Comment by shelegdmitriy** [comment]
> @shelegdmitriy should I merge this PR? let me know in case you're still working on it

No. I'll let you know once I finish it

**Review by charleslavon — APPROVED** [review]
LGTM 👍🏾

File: src/components/HomepageContent/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  );
+}
+
+export default function HomepageContent() {
+  return (
+    <div className="container">
+      <section className="row">
+        <Card
+          Svg={require('@site/static/img/ic_dashboard.svg').default}
+          title="Pagoda PRC"
```
Comment (bucanero) [inline src/components/HomepageContent/index.js]:
```suggestion
          title="Pagoda RPC"
```

**Pull request merged**